src/clients/cassandra/CassandraConnector.js

```diff
@@ -1,19 +1,60 @@
 'use strict';
 
 const Promise = require('promise');
+const cassandra = require('cassandra-driver');
+const asyncEachLimit = require('async/eachLimit');
+const _ = require('lodash/chunk');
```

Comment:
FYI: if we look at the [lodash sources](https://github.com/lodash/lodash/blob/e5e8f35c066c71a04ba584f65acc017d032c0174/chunk.js#L37), the chunk method is exported as a default. If this import is creating problems, you'll have to use `const chunk = require('lodash/chunk').default`. [More info](https://stackoverflow.com/a/35973567/3817588).